src/components/Messages/MessageList/livechatCall.js

```diff
@@ -36,10 +36,17 @@ export const CallNotification = (props) => {
 	const [isframe, setIframe] = useState(false);
 	const [show, setShow] = useState(true);
 
-	const acceptClick = () => {
+	const acceptClick = async () => {
 		setShow(!{ show });
 		if (props.rid.t === 'jitsi_call_started') {
-			window.open(`https://meet.jit.si/xlkhe6qe6dyrlttkx${ props.rid.rid }`);
+			const result = await Livechat.videoCall(props.rid);
```

Comment:
We might need to do some error handling here, incase the `Livechat.videoCall` fails